Don't use old-style function definition

```
compiling tedrahcu/tedrahcu.c
tedrahcu/tedrahcu.c: In function ‘Init_tedrahcu’:
tedrahcu/tedrahcu.c:30:6: warning: old-style function definition [-Wold-style-definition]
   30 | void Init_tedrahcu()
      |      ^~~~~~~~~~~~~
```

--- a/ext/tedrahcu/tedrahcu.c
+++ b/ext/tedrahcu/tedrahcu.c
@@ -27,7 +27,7 @@ static VALUE tedrahcu_detect(VALUE self, VALUE input)
   return result;
 }
 
-void Init_tedrahcu()
+void Init_tedrahcu(void)
 {
   VALUE mRingcurl;
 

